src/Avalonia.Themes.Fluent/Controls/MenuItem.xaml

```diff
@@ -199,6 +198,113 @@
       <Setter Property="IsVisible" Value="False" />
     </Style>
 
+    <Style Selector="^:separator">
+      <Setter Property="Template">
+        <ControlTemplate>
+          <Separator/>
+        </ControlTemplate>
+      </Setter>
+    </Style>
+  </ControlTheme>
+  <ControlTheme x:Key="MobileTextMenuItem" TargetType="MenuItem">
```

Comment:
How much different are these themes to create a completely new one? I think we could reuse the template with BasedOn, and add small style inside to adjust its look.

Reply:
We also have another MenuItem style for Menu (horizontal bar). While I don't expect it to be the same, we definitely can reuse some base styles.